db/redis.go

```diff
@@ -433,11 +440,18 @@ func (r *RedisDriver) getCvesUbuntuWithFixStatus(major, pkgName string, fixStatu
 			}
 			relPatches := []models.UbuntuReleasePatch{}
 			for _, relPatch := range p.ReleasePatches {
-				if relPatch.ReleaseName == codeName {
-					for _, s := range fixStatus {
-						if s == relPatch.Status {
-							relPatches = append(relPatches, relPatch)
-						}
+				found := false
+				for _, codeName := range esmCodeNames {
+					if relPatch.ReleaseName == codeName {
+						found = true
+					}
+				}
+				if !found {
+					continue
+				}
+				for _, s := range fixStatus {
+					if s == relPatch.Status {
+						relPatches = append(relPatches, relPatch)
 					}
 				}
 			}
```

Comment:
It could be refactored like this.
```suggestion
				if slices.Contains(esmCodeNames, relPatch.ReleaseName) && slices.Contains(fixStatus, relPatch.Status) {
					relPatches = append(relPatches, relPatch)
				}
			}
```